README.md

```diff
@@ -29,9 +29,9 @@ Or install it yourself as:
 ```ruby
 class CalculationService < BoringService
   # Type checking is optional. If omitted, anything is accepted.
-  # Type checking can also be done with a proc or anything that responds to #===
+  # Type checking can also be done with a proc or anything that responds to #=== (and even an array of it!)
   # e.g. parameter :start_number, ->(p) { p.respond_to?(:to_i) }
-  parameter :start_number, Integer
+  parameter :start_number, [Integer, Float], required: true
```

Comment:
```suggestion
  parameter :start_number, Integer
  parameter :start_number_or_string, [Integer, String]
```

Reply:
And then the rest of the example below would need to reflect the addition of the param that can be either a number or a string.

Reply:
And I removed the `required: true`, since that's not a param.

Reply:
> And I removed the required: true, since that's not a param.

actually, you're showing an example below where you raise a `BoringService::ParameterRequired` error when calling the `CalculationService` without `start_number`. I was actually fixing your test class in the doc to properly reflex the documentation examples 🤔 

<img width="818" alt="image" src="https://github.com/user-attachments/assets/a2242f28-511f-4d69-9922-f09d7717dbd6">

or am I missing something @jeffdill2 ?

Reply:
It functions the same way as vanilla Ruby (intentionally). I.e. if you define a parameter, you must provide it, unless you define a default value, in which case you don't have to provide it.